tfx/components/transform/executor_test.py

```diff
@@ -58,9 +60,25 @@ class _TempPath(types.Artifact):
 # TODO(b/122478841): Add more detailed tests.
 class ExecutorTest(tft_unit.TransformTestCase):
 
-  def _get_source_data_dir(self):
-    return os.path.join(
+  @classmethod
+  def setUpClass(cls):
+    cls._temp_example_dir = tempfile.mkdtemp()
```

Comment:
Could you move all of these cls._... = ... to just be top level in the class?
So only the things that can't be static such as copy_dir, glob, etc. are done here.
Also, note that class level constants should be named with all caps.